### PR TITLE
Re-skin the rest of the bounce levels as basketball levels

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

**Comment by balderdash** [comment]
At least for the levels.js changes, you'll want to view the diff [with whitepsace ignored](https://github.com/code-dot-org/code-dot-org/pull/11412/files?w=1).

**Comment by balderdash** [comment]
Circle is running out of memory, doesn't look like it has any real failures.

**Comment by Hamms** [comment]
Yeah, that's been a recurring problem. If you're worried about tests, you can add `[test chrome]` to a commit message to switch things back to saucelabs

File: apps/src/bounce/levels.js [hunk 1]
```diff
@@ -1,219 +1,261 @@
...
+    [8, 0, 0,16, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0]
+  ],
+  'toolbox':
+    tb('<block type="bounce_moveLeft"></block> \
+        <block type="bounce_moveRight"></block>'),
+  'startBlocks':
+   '<block type="bounce_whenLeft" deletable="false" x="20" y="20"></block>'
+};
+levels['1_basketball'] = extend(levels['1'], {
```
Comment (Hamms) [inline apps/src/bounce/levels.js:36]: :+1:

**Comment by balderdash** [comment]
Circle failure is unrelated.

**Pull request merged**